code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -96,8 +96,8 @@
 	foldable = null //special form fitted boxes should not be foldable.
 
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "personal emergency kit"
+	desc = "A survival box containing a variety of supplies. This one contains oxygen, as shown by the red panel and two striped yellow lines."
```

Comment:
```suggestion
	desc = "A survival box containing a variety of supplies. The red panel and two striped yellow lines indicate this one contains oxygen."
```

Reply:
Wordy wording overly wording basis wording- ok.

Reply:
Not resolved, your wording is overly wordy please write what I requested.

Reply:
But- I did!

Reply:
Oh yeah you did idk why github bugged out and showed me the old one